test/jdk/ProblemList.txt

```diff
@@ -585,7 +585,7 @@ sun/security/pkcs11/sslecc/ClientJSSEServerJSSE.java            8161536 generic-
 
 sun/security/tools/keytool/ListKeychainStore.sh                 8156889 macosx-all
 
-javax/net/ssl/SSLEngine/TestAllSuites.java                      8298874 generic-all
+javax/net/ssl/SSLEngine/IllegalRecordVersion.java               8298873 generic-all
```

Comment:
you will need to update to latest source, this test was removed from ProblemList.